### PR TITLE
Support for restart dev mode containers

### LINKED ISSUE #269: Support for "okteto restart"
**Is your feature request related to a problem? Please describe.**
I want an easy way to restart my dev pods.

**Describe the solution you'd like**
"okteto restart" restarts the pods of my dev environment.

### PR DESCRIPTION
**Pull request by pchico83**
Closes #269

### CODE REVIEW

File: cmd/restart.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+//Restart restarts the pods of a given dev mode deployment
+func Restart() *cobra.Command {
+	var namespace string
+	// get the name from it
+	var devPath string
+
+	cmd := &cobra.Command{
+		Use:   "restart COMMAND",
+		Short: "Restarts the pods of a Okteto Environment",
```
Comment (rberrelleza) [inline cmd/restart.go]:
```suggestion
		Short: "Restarts the pods of your Okteto Environment",
```

File: cmd/restart.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+)
+
+//Restart restarts the pods of a given dev mode deployment
+func Restart() *cobra.Command {
+	var namespace string
+	// get the name from it
+	var devPath string
+
+	cmd := &cobra.Command{
+		Use:   "restart COMMAND",
```
Comment (rberrelleza) [inline cmd/restart.go]: Do we need to pass a command with restart? if not, it should just be the command name

```suggestion
		Use:   "restart",
```

File: cmd/restart.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+	if dev.Namespace == "" {
+		dev.Namespace = namespace
+	}
+
+	restarted := map[string]bool{}
+	for _, s := range dev.Services {
+		if _, ok := restarted[s.Name]; ok {
+			continue
+		}
+		if err := pods.Restart(s.Name, dev.Namespace, client); err != nil {
```
Comment (rberrelleza) [inline cmd/restart.go]: is this waiting for the restart to finish? if so, maybe do it in a goroutine so we parallelize it?

Comment (pchico83) [reply cmd/restart.go]: No, it does not wait.

**Pull request merged**